src/portal/src/app/base/harbor-shell/harbor-shell.component.scss

```diff
@@ -50,3 +50,38 @@ clr-vertical-nav {
 	}
 }
 
+.scanner-info {
+    margin-right: 0;
+    margin-left: 0;
+    background-color: #0079b8;
+    height: 2rem;
+    color: #fff;
+    align-items: center;
+    font-size: 0.5rem;
+    a {
+        cursor: pointer;
+        text-decoration: #bababa underline;
+        font-weight: 600;
+    }
+    a:hover {
+        color: #fff;
+    }
+    a:visited {
+        color: #fff;
+    }
+    a:link {
+        color: #fff;
+    }
```

Comment:
a:hover {
        color: #fff;
    }
    a:visited {
        color: #fff;
    }
    a:link {
        color: #fff;
    }
Don't repeat yourself in the code. 
Please use:
    a:hover,a:visited,a:link {
        color: #fff;
    }

Reply:
fixed